### ISSUE TITLE
Tutorial 1 crashes on Windows

### ISSUE THREAD

**Issue by prock-fife** [fifengine/cpp-tutorials#15]
The tutorial needs to be updated to work with FIFE master branch.  It crashes right after you run it.

**Comment by vtchill** [comment]
I'll see if i can take a look soon. I'll be gone for a couple days, but will try when i get back later next week.

**Comment by DnJealt** [comment]
I've tried the same with a fresh install of FIFE.. Same result: crashes on startup.
What can I do to help solve this issue?

**Comment by LinuxDonald** [comment]
On linux the tut1 runs fine. It must be an problem with windows code?

**Comment by milandamen** [comment]
Indeed, on Linux it runs fine. The tutorial is also limited to only Release on VS14 on Windows.
